pcdsdevices/epics_motor.py

```diff
@@ -578,7 +578,14 @@ class IMS(PCDSMotorBase):
     velocity_base = Cpt(EpicsSignal, '.VBAS', kind='omitted')
     velocity_max = Cpt(EpicsSignal, '.VMAX', kind='config')
 
-    tab_whitelist = ['reinitialize', 'acceleration', 'clear_.*']
+    tab_whitelist = [
+        'reinitialize',
+        'acceleration',
+        'clear_.*',
```

Comment:
While I like the `.*` regex for coding convenience (or laziness, but no judgment implied!), I think it's better to be explicit in these.